### PR TITLE
feat: bootstrap task

### PR DESCRIPTION
**Pull request by bastienboutonnet**
# Description
Adds a new command called `bootstrap` which allows dbt users to generate placeholders for all models as well as their columns in a dbt project.

This command can be ran before doing an `audit` to generate placeholders for all models so that the results of the `audit` task (which relies solely on content present in `schema.yml` files) are exhaustive and accurate.

# How was the change tested

# Issue Information
Partially addresses #210

# Checklist

- [x] I formatted my PR name according to [CONTRIBUTING.md](CONTRIBUTING.md)
- [x] I added a news fragment to help populating the changelog as encouraged in [CONTRIBUTING.md](CONTRIBUTING.md)
- [x] I added "Closes #<issue_number>" in the "Issue Information" section (if no issue, feel free to tick thick the box anyway).

### CODE REVIEW

File: changelog/233.feature.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dbt-sugar can now automatically generate bootstrap model descriptor files (schema.yml) for all your models as well as their columns. Bootstrap model descriptors will contain eithe placeholders for undocumented columns and model descrptions, unless columns are documented in other models and in that case `bootstrap` will populate those columns with their definitions.
+
```
Comment (virvirlopez) [inline changelog/233.feature.md:2]: typo in:
eithe
descrptions

File: dbt_sugar/core/connectors/base.py [hunk 2]
```diff
@@ -30,10 +30,8 @@ def __init__(
         self.engine = sqlalchemy.create_engine(**connection_params)
 
     def get_columns_from_table(
-        self,
-        target_table: str,
-        target_schema: str,
-    ) -> Optional[List[Tuple[Any]]]:
+        self, target_table: str, target_schema: str, **kwargs
```
Comment (virvirlopez) [inline dbt_sugar/core/connectors/base.py:33]: what is this argument **kwargs
is missing in the documentation

Comment (bastienboutonnet) [reply dbt_sugar/core/connectors/base.py:33]: It's for the snowflake connector which may need to get the argument for `use_describe`. How would you document it?

File: changelog/233.feature.md (hunk 1 above)
Comment (bastienboutonnet) [reply changelog/233.feature.md:2]: good catch will fix it now

**Review by virvirlopez — DISMISSED** [review]
LGTM

**Pull request merged**